src/deployment/fabricProvisioningHelpers.ts

```diff
@@ -789,10 +759,25 @@ export function sendFabricProvisioningCloseEventTelemetry(state: fp.FabricProvis
     );
 }
 
-export function updatefabricProvisioningState(
+export function updateFabricProvisioningState(
     deploymentController: DeploymentWebviewController,
     newState: fp.FabricProvisioningState,
 ) {
     deploymentController.state.deploymentTypeState = newState;
     deploymentController.updateState(deploymentController.state);
 }
+
+export async function getTenantOptions(accountId: string): Promise<FormItemOptions[]> {
+    const tenants = await VsCodeAzureHelper.getTenantsForAccount(accountId);
+
+    return tenants

```

Comment:
Can you add a coment explaining how the list is getting sorted?  It looks like it's moving any tenant named "Microsoft" to the top of the list, but I'm not sure why ('cause that would only really benefit Microsoft employees, right?)